### ISSUE TITLE
Does not work when using NVDA+Down-Arrow

### ISSUE THREAD

**Issue by jooyoungseo** [mltony/nvda-phonetic-punctuation#3]
Phonetic Punctuation and Audio Rules are ignored when reading from system caret (NVDA+Down-Arrow). Could you please make the audio rules also work for this situation?

Many thanks!
